File: Hydrogene/All/SharedUI/SharedUI.Shared/AppDelegate.Cocoa.cs
namespace SharedUI.Shared
{
	#if TOFFEE
	[NSApplicationMain, IBObject]
	public partial class AppDelegate : INSApplicationDelegate
	{
		public void applicationDidFinishLaunching(NSNotification notification)
		{
			start();
		}

		//
		// Add Cocoa-specific code here
		//
	}
	#endif
}

File: Iodine/All/SharedUI/SharedUI.Shared/AppDelegate.Cocoa.java
package SharedUI.Shared;

#ifdef TOFFEE

@NSApplicationMain, @IBObject
public __partial class AppDelegate extends INSApplicationDelegate
{
	public void applicationDidFinishLaunching(NSNotification notification)
	{
		start();
	}

	//
	// Add Cocoa-specific code here
	//
}

#endif